frame/dapps-staking/src/traits.rs

```diff
@@ -0,0 +1,3 @@
+pub trait IsContract {
+    fn is_contract(&self) -> bool;
```

Comment:
Imo, this should just be in the mod.rs of the pallet.

Reply:
there is no fast rule on where the traits should be defined. We can do it either way

Reply:
I commented since it's 3 lines of code.
You could re-export it as something from pallet_dapps_staking::trait if you want to use trait.

But it's fine like this too.

Now that I wrote the comment above, I realized there's no documentation on the method :P